public/js/login.js

```diff
@@ -0,0 +1,32 @@
+const loginFormHandler = async (event) => {
+  event.preventDefault();
+
+  // trim = leerzeichen vorne und hinten abschneiden
+  const username = document.querySelector("#username").value.trim();
+  const password = document.querySelector("#password").value.trim();
+
+  if (username && password) {
```

Comment:
fantastic initative to only continue if the username and password have been provided! :-)
Consider adding and else statement here to e.g. display an error message in case this condition is not fulfilled.